api.md

```diff
@@ -260,7 +260,7 @@ error or a notice depending on the value of `if_not_exists`.
 
 If `bootstrap` is true, the function will attempt to bootstrap the
 data node by:
-1. Connecting to the database given in `bootstrap_database`.
+1. Connecting to the PostgreSQL maintance database (`postgres` or `template1`).
```

Comment:
What is a "maintenance" database?  This isn't a term we used before, and sounds like a new type of organizational role which seems confusing / not sure if we want to use?

(Regardless, spelling issue:
```suggestion
1. Connecting to the PostgreSQL mainetance database (`postgres` or `template1`).
```

Reply:
I believe "maintenance" is the PostgreSQL terminology, at least they are using it in the source code:  https://github.com/postgres/postgres/blob/master/src/bin/scripts/common.c#L173.

It serves bootstrapping purposes. For example when you need to create a database your client application (libpq/psql) needs to connect to some database first in order to execute `CREATE DATABASE` command.

Reply:
I think we could also remove the first step completely